geode-core/src/test/java/org/apache/geode/test/dunit/internal/JUnit4DistributedTestCase.java

```diff
@@ -91,7 +91,7 @@
   private static final Set<String> testHistory = new LinkedHashSet<>();
 
   /** This VM's connection to the distributed system */
-  private static InternalDistributedSystem system;
+  protected static InternalDistributedSystem system;
```

Comment:
I don't see the code actually using this field?

Reply:
It's on line 345 of the test